account_invoice_search_by_reference/model/account_invoice.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
Thanks for migrating this PR. Could you change the copyright please? Now it is

`Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)`

Reply:
@AaronHForgeFlow 
Edited, Thank you for your review.